### PR TITLE
Fix vulnerabilities in GradleV3, up shalljs version till 0.8.5

### PR DESCRIPTION
**Pull request by SvetlanaMaliugina**
**Task name**: GradleV3

**Description**:
- Fixed minor vulnerabilities by running the ```npm audit fix``` command.
- ```azure-pipelines-task-lib``` version was updated too because the previous version use ```shalljs 0.8.4``` 

**Documentation changes required:** N

**Added unit tests:** N

**Attached related issue:** [[Governance alerts] GradleV3 task (shelljs) #3068](https://github.com/microsoft/build-task-team/issues/3068)

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected ( run unit tests and test on canary pipeline)

### CODE REVIEW

**Review by vmapetr — APPROVED** [review]

File: Tasks/GradleV3/package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   "dependencies": {
     "@types/node": "^10.17.0",
     "@types/mocha": "^5.2.7",
-    "azure-pipelines-task-lib": "^3.1.6",
+    "azure-pipelines-task-lib": "3.2.0",
```
Comment (DenisRumyantsev) [inline Tasks/GradleV3/package.json:13]:
```suggestion
    "azure-pipelines-task-lib": "^3.3.1",
```

**Review by DenisRumyantsev — APPROVED** [review]

**Comment by SvetlanaMaliugina** [comment]
@mmrazik could you please approve it?

**Review by alexander-smolyakov — APPROVED** [review]
LGTM, thanks! Could you please ask someone from the team to perform buddy-testing? And please post the results of the risks analysis checklist.

**Comment by DmitriiBobreshev** [comment]
performed buddy testing for task:
- Unit Testing Result: all tests passed, file: [GradleV3TestsOut.txt](https://github.com/microsoft/azure-pipelines-tasks/files/8964906/GradleV3TestsOut.txt)
- Pipeline Tests on Win, Mac and Linux machines, tests passed, logs: [GradleV3TestsPipeline.zip](https://github.com/microsoft/azure-pipelines-tasks/files/8964907/GradleV3TestsPipeline.zip)

**Pull request closed without merging**